mmv1/third_party/terraform/services/netapp/resource_netapp_active_directory_sweeper.go

```diff
@@ -0,0 +1,128 @@
+package netapp
+
```

Comment:
looks like this file has some lint fails, could you address them in this file? Related gha: https://github.com/GoogleCloudPlatform/magic-modules/actions/runs/10535926639/job/29195530738

Reply:
Done